docs: Refactor condition messages and reasons (#1099)
kyma-project/telemetry-manager: docs/user/04-metrics.md

Set up scraping through HTTP by applying the `prometheus.io/scheme=http` annotation.

<!-- tabs:end -->

### Buffer Filling Up

>   Cause: The backend sizing is inadequate

Reviewer: ```suggestion
  Cause: The backend sizing is too small for the data influx.
```

  Remedy:
  

>   - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)

Reviewer: ```suggestion
  - Option 1: Increase ingestion rate capabilities in your backend. For example, scale out the SAP Cloud Logging or Dynatrace instances.
```

>   - Option 2: Decrease data influx (i.e. re-configure the metric pipeline)

Reviewer: ```suggestion
  - Option 2: Decrease data influx, that is, reconfigure the metric pipeline.
```

### Gateway Throttling 

>   Cause: Gateway is unable to receive metrics at the given rate

Reviewer: ```suggestion
  Cause: Gateway cannot to receive metrics at the given rate.
```

  Remedy:

>   - Manually scale out the gateway (i.e. increase the number of replicas for the `telemetry-metric-gateway`). See [Module Configuration](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration)

Reviewer: ```suggestion
  - Manually scale out the gateway by increasing the number of replicas for the `telemetry-metric-gateway`. See [Module Configuration](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
```
